### PR TITLE
Cleaned up C++ project targets

### PR DESCRIPTION
**Pull request by mattscheffer**
@tmeschter Here's the changes.

### CODE REVIEW

File: src/Compilers/Core/NativeClientTests/NativeClientTests.vcxproj.filters [hunk 1]
```diff
@@ -28,5 +28,6 @@
     <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
     <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
     <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
+    <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
```
Comment (gafter) [inline src/Compilers/Core/NativeClientTests/NativeClientTests.vcxproj.filters]: Why?

File: src/Compilers/VisualBasic/vbc2/vbc2.vcxproj.filters [hunk 2]
```diff
@@ -34,5 +34,6 @@
       <Filter>Resource Files</Filter>
     </ResourceCompile>
     <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
+    <ResourceCompile Include="$(GeneratedVersionInfoResourceFile)" />
```
Comment (gafter) [inline src/Compilers/VisualBasic/vbc2/vbc2.vcxproj.filters]: Why?

Comment (tmeschter) [inline src/Compilers/VisualBasic/vbc2/vbc2.vcxproj.filters]: @gafter: This seems to be an accidental additional by the VS tooling. @mattscheffer is looking into removing it.

**Comment by gafter** [comment]
:+1:

**Pull request merged**